source/_components/plant.markdown

```diff
@@ -27,9 +32,42 @@ plant:
     min_temperature: 15
 ```
 
-## Using plain MQTT sensor to get the data
+Configuration variables:
+
+- **simulated_plant** (*Required*):
+  - **sensors** (*Required*): 
+    - **moisture** (*Optional*):  
+    - **battery** (*Optional*):
+    - **temperature:** (*Optional*):
+    - **conductivity:** (*Optional*):
+    - **brightness:** (*Optional*):
+  - **min_moisture** (*Optional*):
+  - **max_moisture** (*Optional*):
+  - **min_battery** (*Optional*):
+  - **min_conductivity** (*Optional*):
+  - **min_temperature** (*Optional*):
+
+## {% linkable_title Examples %}
+### Using plain MQTT sensor to get the data
+This is a practial example that uses a multiple of `MQTT sensors` to supply the readings used by the `plant` sensor.
+
```

Comment:
Maybe add a reference to https://home-assistant.io/components/sensor.miflora/ since it is the sensor that delivers all those measurements. If we have more sensors in the future we can add them as well.